### PR TITLE
NEW: use Expedition::$date_shipping property in card and list

### PR DESCRIPTION
**Pull request by altairis-noe**
# NEW use Expedition::$date_shipping property in card and list
Currently, Expedition object does have a `$date_shipping` property, stored as `date_expedition` in the database.

But there is no way in Dolibarr interface to set or display its value.

If your customer wants to know when he will receive the goods, you might want to know when it is gone from your facilities. Here is my proposition.

image

image

image

### CODE REVIEW

File: htdocs/expedition/card.php [hunk 1]
```diff
@@ -268,6 +270,7 @@
 		$object->ref_customer = GETPOST('ref_customer', 'alpha');
 		$object->model_pdf = GETPOST('model');
 		$object->date_delivery = $date_delivery; // Date delivery planned
+		$object->date_shipping = $date_shipping; // Date delivery planned
```
Comment (sonikf) [inline htdocs/expedition/card.php:273]: Hi @altairis-noe 
Please change the comment to // Sending date

Comment (altairis-noe) [reply htdocs/expedition/card.php]: Thanks for the review @sonikf. It’s done!

Comment (sonikf) [reply htdocs/expedition/card.php]: https://github.com/Dolibarr/dolibarr/blob/71cde0e95f8131b92edc7e25c49b5d7ecc890902/htdocs/compta/facture/class/facture.class.php#L127
Can you please change this comment too as it is misleading and makes no sense, planned and expected date has nothing to do with actual date of start

File: htdocs/expedition/card.php [hunk 2]
```diff
@@ -1030,6 +1033,15 @@
 				print '</tr>';
 			}
 
+			// Date sending
```
[the remark this answers is not in the document]
Comment (altairis-noe) [reply htdocs/expedition/card.php]: Hey @eldy,

No problems for me, it’s done!
image

**Comment by altairis-noe** [comment]
Hello @eldy,

is there still something blocking the merging of that one?

**Pull request merged**